src/kernel.cr

```diff
@@ -1,18 +1,39 @@
 {% if flag?(:win32) %}
+  # The standard input file descriptor.
   STDIN  = IO::FileDescriptor.new(0)
+  # The standard output file descriptor. Gets flushed when a newline is written.
```

Comment:
I don't think `NOTE` is needed here.
It's already pretty much 2 times stated here with the code `{ |f| f.flush_on_newline = true }` that the file descriptor gets flushed on a newline.
And now since the last commit it would look like this:
```cr
# The standard output file descriptor.
#
# Typically used to output data and information.
#
# NOTE: Gets flushed when a newline is written.
```
I think that's just unnecessarily long and can be shorter.

Reply:
I'd say it fits better as a `NOTE` since it's not in any way description of a constant but rather a _note_ about its behaviour. And it's not longer since `NOTE` is shown differently (and more prominently) in docs.